exercises/02-publish-subscribe/README.md

```diff
@@ -73,14 +73,14 @@ namespace Divergent.Sales.Messages.Events
 
 ### Step 3
 
-Have a look at `SubmitOrderHandler` class in the `Divergent.Sales` project. At the end of the `Handle` method, publish the `OrderSubmittedEvent`, by calling the ```context.Publish<OrderSubmittedEvent>()``` method. Copy the properties from the incoming `SubmitOrderCommand` message, to the properties of the event.
+Have a look at `SubmitOrderHandler` class in the `Divergent.Sales` project. At the end of the `Handle` method, publish the `OrderSubmittedEvent`, by calling the ```context.Publish()``` method. Copy the properties from the incoming `SubmitOrderCommand` message, to the properties of the event.
 
 ```c#
-await context.Publish<OrderSubmittedEvent>(e =>
+await context.Publish(new OrderSubmittedEvent
 {
-    e.OrderId = order.Id;
-    e.CustomerId = message.CustomerId;
-    e.Products = message.Products;
+    OrderId = order.Id;
```

Comment:
You still have semicolons in here

Reply:
Oops! Good catch. Since the change was so trivial, I amended the commit directly. Should be good now.